charts/portworx/values.yaml

```diff
@@ -101,6 +101,15 @@ tolerations:                          # Add tolerations
   #   value: "value"
   #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
 
+nodeAffinity:
```

Comment:
Do we support only nodeAffinity in stc?

Reply:
No @diptiranjanpx we have support for both nodeAffinity and tolerations in stc.
https://docs.portworx.com/portworx-enterprise/reference/crd/storage-cluster#placement-rules